src/components/form.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+function Form() {
+  <div>
+    <h3 className="form-book-title">Add New Book</h3>
+    <form>
+      <input type="text" placeholder="Book Title" />
+      <input type="text" placeholder="Author" />
+      <button type="button" className="button" id="addBtn">
+        ADD BOOK
+      </button>
+    </form>
+  </div>;
+}
```

Comment:
- [ ] Nice work implementing the books list :clap: However the form to add new book is not showing in the browser (see image). And that it's because the component `form.jsx` is not returning anything. Kindly make sure your form component it returning a form to add new book.
![Screenshot from 2023-08-18 08-32-18](https://github.com/mhnazary/bookstore/assets/91142432/996575a9-6d6e-46d7-85b9-26975c4ed805)

**HIT: This component could look like this :point_down:**
```js
  return (
    <div>
      <h3 className="form-book-title">Add New Book</h3>
      <form>
        <input type="text" placeholder="Book Title" />
        <input type="text" placeholder="Author" />
        <button type="button" className="button" id="addBtn">
          ADD BOOK
        </button>
      </form>
    </div>
  );
```